roles/edpm_nova/tasks/configure.yml

```diff
@@ -101,6 +85,51 @@
   tags:
     - edpm_users
 
+- name: Sync nova config files
+  become: true
+  tags:
+    - configure
+    - nova
+  ansible.posix.synchronize:
+    src: "{{ edpm_nova_config_sync_src }}"
+    dest: "{{ edpm_nova_config_dest }}"
+    recursive: true
+    delete: true
+    mode: "push"
+    owner: false
+    group: false
+  notify:
+    - Restart nova
+
+- name: Render nova config files
+  become: true
```

Comment:
as we sudo it, let;s specify the owner and user?

Reply:
this should be ok I think